GWAS/LMM.ipynb

```diff
@@ -1350,12 +1501,12 @@
     "    ifelse((${ylim} == 0 && min(data$${pval}, na.rm=TRUE)!=0), ylim <- abs(floor(log10(min(data$${pval}, na.rm=TRUE)))), ylim <- abs(floor(log10(2.225074e-308))))\n",
     "    # Creating manhattan plot\n",
     "    png('${_output[0]}', width = 6, height = 4, unit='in', res=300)\n",
-    "    manhattan_plot <- manhattan(data, chr='CHR', bp='${bp}', snp='${snp}', p='${pval}', main = 'Manhattan plot for ${_phenoCol} (${step_name.rsplit(\"_\",1)[0]})', ylim = c(0, ylim), cex = 0.6, \n",
+    "    manhattan_plot <- manhattan(data, chr='CHR', bp='${bp}', snp='${snp}', p='${pval}', main = 'Manhattan plot for ${_phenoCol} (${step_name.rsplit(\"_\",1)[0]})', ylim = c(0, ${ylim}), cex = 0.6, \n",
```

Comment:
See line 1501 we actually adjusted `ylim` to a variable locally defined. It is thus not correct to use `${ylim}` in this line, right? Unless you have otther reasons I dont see?